test/plugin/base_test.rb

```diff
@@ -2387,8 +2569,13 @@ def verify_cascading_json_detection_with_log_entry_fields(
     end
   end
 
-  def verify_field_key(log_entry_field, default_key, custom_key,
-                       custom_key_config, sample_value)
+  def verify_field_key(log_entry_field, test_params)
+    default_key = test_params[:default_key]
```

Comment:
I'm not too crazy about this, but we can clean it up later.

Reply:
Cool